### PR TITLE
Completed challenge!

### PR DESCRIPTION
**Pull request by loyslane**

### CODE REVIEW

**Comment by loyslane** [comment]
completed operators challenge

File: array/array.js [hunk 1]
```diff
@@ -7,3 +7,78 @@
...
+      newNode.index = this.length;
+      this.length++;
+    } else if (this.length === 1) {
+      this.head.next = newNode;
+      newNode.previous = this.head;
+      this.tail = newNode;
+      newNode.index = this.length;
+      this.length++;
+    } else {
+    this.tail.next = newNode;
```
Comment (ryan-hamblin) [inline array/array.js:36]: indentation here is a little off.

File: array/array.js [hunk 2]
```diff
@@ -7,3 +7,78 @@
...
  */
+
+class Array {
+  constructor() {
+    this.head = null;
+    this.tail = null;
+    this.length = 0;
+  }
+  push(element) {
+    const newNode = {
```
Comment (ryan-hamblin) [inline array/array.js:18]: Great idea on using a linked list here. I've seen this done in this way before. There are more concise and simpler solutions out there, and i recommend looking at those too! But I LOVE this idea of piggy backing off your previous knowledge.

**Review by ryan-hamblin — COMMENTED** [review]
Looks like a bit of inconsistancy across your indentation in your files. If you just installed VSCode I recommend researching how you can change your indentation to be 2 spaces instead of 4. cause.. it's pretty.

**Pull request closed without merging**